spk/synocli-net/Makefile

```diff
@@ -1,24 +1,19 @@
 SPK_NAME = synocli-net
 SPK_VERS = 1.6
```

Comment:
IMHO we need to change the main version, as we have a breaking change (for users of fritzctrl).
SPK_VERS = 2.0

Reply:
Fixed in last commit.